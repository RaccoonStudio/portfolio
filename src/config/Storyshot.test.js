import initStoryshots from '@storybook/addon-storyshots';
import { imageSnapshot } from '@storybook/addon-storyshots-puppeteer';

const customizePage = page => {
  return page.emulate({
    'name': 'Desktop 1920x1080',
    'userAgent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.75 Safari/537.36',
    'viewport': {
      'width': 1920,
      'height': 1080,
      'deviceScaleFactor': 2,
      'isMobile': false,
      'hasTouch': false,
      'isLandscape': true
    }
  });
};

initStoryshots({ suite: 'Image storyshots', test: imageSnapshot({ customizePage }) });
