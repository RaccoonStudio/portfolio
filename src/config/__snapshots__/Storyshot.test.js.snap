// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots PageTitle overview 1`] = `
<h1
  className="sc-bdVaJa iTWoZQ"
>
  Hello World!
</h1>
`;
